woocommerce-square.php

```diff
@@ -22,8 +22,8 @@
  * @copyright Copyright (c) 2019, Automattic, Inc.
  * @license   http://www.gnu.org/licenses/gpl-3.0.html GNU General Public License v3.0 or later
  *
- * WC requires at least: 9.0
- * WC tested up to: 9.2
+ * WC requires at least: 9.3
+ * WC tested up to: 9.1
```

Comment:
These need to be swapped, as the tested up to should be 9.3 and the minimum should be 9.1

Reply:
Done 👍